### PR TITLE
lisp-packages.nix: update Nyxt

### PR DESCRIPTION
**Pull request by jmercouris**
@7c6f434c 

These are the changes I had to make to update Nyxt for NixOS.

Closes #127906

### CODE REVIEW

**Comment by 7c6f434c** [comment]
Hmmm, so you need CL-WebKit2 fresher than the latest Quicklisp? Or would it be enough to run Quicklisp update?

(also, I think you said you expected an extra package to be needed)

**Comment by jmercouris** [comment]
We sometimes need cl-webkit2 fresher than the latest quicklisp. We need extra packages for 2.2.0, and 2.3.0.

File: pkgs/development/lisp-modules/lisp-packages.nix [hunk 1]
```diff
@@ -194,7 +194,7 @@ let lispPackages = rec {
       owner = "atlas-engineer";
       repo = "nyxt";
       rev = "${version}";
-      sha256 = "sha256-eSRNfzkAzGTorLjdHo1LQEKLx4ASdv3RGXIFZ5WFIXk=";
+      sha256 = "sha256:0kzm05swhyb197cjfd3iglf60b997sx7v95yxzyq483jxqbcxm0r";
```
Comment (SuperSandro2000) [inline pkgs/development/lisp-modules/lisp-packages.nix:197]:
```suggestion
      sha256 = "0kzm05swhyb197cjfd3iglf60b997sx7v95yxzyq483jxqbcxm0r";
```

Comment (jmercouris) [reply pkgs/development/lisp-modules/lisp-packages.nix:197]: ???

Comment (7c6f434c) [reply pkgs/development/lisp-modules/lisp-packages.nix:197]: I think the long-term plan is SRI hashes everywhere, so moving away from SRI seems not necessary?.

**Comment by 7c6f434c** [comment]
I am not exactly eager to edit generated files, and putting the real version into overrides sounds «update-forgetting-prone»… So I guess globally the way to go is to add cl-webkit2-nyxt in lispPackages that overrides the Quicklisp-converted version.

**Comment by jmercouris** [comment]
Perhaps that is the solution, in any case, I don't know how to make it happen. When 2.3.0 comes out I will ping you with the new dependencies and then we could do it at that time. I think that makes the most sense. What do you think?

**Comment by r-rmcgibbo** [comment]
Result of `nixpkgs-review pr 131665` at c74db90f run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>3 packages built successfully:</summary>
  <ul>
    <li>lispPackages.cl-webkit2</li>
    <li>lispPackages.nyxt</li>
    <li>nyxt</li>
  </ul>
</details>

File: pkgs/development/lisp-modules/lisp-packages.nix (hunk 1 above)
Comment (jmercouris) [reply pkgs/development/lisp-modules/lisp-packages.nix:197]: I simply pasted what Nix reported to me on error. If I should not have "sha256:" in the string, that is fine with me. I don't see the difference really.

Comment (SuperSandro2000) [reply pkgs/development/lisp-modules/lisp-packages.nix:197]: None sri hashes should not have the prefix. SRI hashes are allowed to have them.

Comment (7c6f434c) [reply pkgs/development/lisp-modules/lisp-packages.nix]: Wait, what is «allowed» here? They won't work without it if you mean `hash = `, and they won't be in SRI format without prefix in `sha256 =` case

**Comment by 7c6f434c** [comment]
A proper Quicklisp update has happened since, and together with Nyxt update to a fresher version

**Pull request closed without merging**